src/SilexMemcache/MemcacheExtension.php

```diff
@@ -1,43 +1,51 @@
 <?php
 
-
 namespace SilexMemcache;
 
 use Silex\Application;
 use Silex\ServiceProviderInterface;
+use Memcache;
+use Memcached;
 
 
 class MemcacheExtension implements ServiceProviderInterface
 {
+
+    const MEMCACHE = 'memcache';
+
+    const MEMCACHED = 'memcached';
+
     public function boot(Application $app)
     {
-
+        
     }
 
     public function register(Application $app)
-    {  
+    {
         $app['memcache'] = $app->share(function () use ($app) {
-            
-            $library = isset($app['memcache.library']) ? strtolower($app['memcache.library']) : 'memcached';
+            $library = isset($app['memcache.library']) ? strtolower(trim($app['memcache.library'])) : self::MEMCACHED;
             $servers = isset($app['memcache.server']) ? $app['memcache.server'] : array(
-                array('127.0.0.1', 11211)     
+                array('127.0.0.1', 11211)
             );
-            
```

Comment:
That'd be preferable for me. With the new code there is an explicit check against the constants. If we add a new driver we need to update two locations rather than just one when we used an `else` statement.

If you change the PR to use an else again rather than checking up front I can merge in